### PR TITLE
Added support for YINMIK YK-S03 Pool Monitoring

### PR DESCRIPTION
**Pull request by lauer**
This will add support for YINMIK YK-S03 Smart pH and Chlorine Tester for Swimming Pool

https://www.amazon.com/YINMIK-Chlorine-Swimming-Salinity-Inground/dp/B0C2T8YLYW

It look like the same device as BLE-YL01, but some params might be a bit different.
Koenkk/zigbee2mqtt#18704

Image added in this PR Koenkk/zigbee2mqtt.io#2796

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -8712,6 +8712,76 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_d9mzkhoq']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:8716]: I propose to merge this definition with BLE-YL01 (add the missing features there, probably they are exactly the same devices). For the correct detection, use `tuya.whitelabel` (search in this file for examples)

Comment (lauer) [reply src/devices/tuya.ts:8716]: What about the image? They looks a bit different.

Comment (Koenkk) [reply src/devices/tuya.ts:8716]: It will show in the UI with the props defined in the whitelabel, including the picture.

Comment (lauer) [reply src/devices/tuya.ts]: I have merged the definition now, and added some whitelabels, as how I understod it works.

Is it okay now?

**Comment by Koenkk** [comment]
Perfect, thanks!

**Pull request merged**